backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/MigrateVmCommand.java

```diff
@@ -613,7 +622,10 @@ public void runningSucceded() {
             updateVmAfterMigrationToDifferentCluster();
             plugPassthroughNics();
             initParametersForExternalNetworks(destinationVds, true);
+            setDedicatedCpus();
         } finally {
+            // clear source VDS taken resources
+            super.getVdsManager().unpinVmCpus(getVmId());
```

Comment:
might be possible to remove, since we clear on DestroyVDS. The benefit here is: 1. safety. 2. resources release faster.

Reply:
I agree with Lucia's comment below - the use of super.getVdsManager is confusing, let's be more explicit about when we refer to the source host and when to the destination host. I'm ok with keeping it, the part in destroy-vm is more bothering - would be great if we can avoid it there and do that when we get the VM as Down by the monitoring